src/data_operations.py

```
@@ -5,35 +5,49 @@
-    sofr_data = pd.DataFrame()
-    audit_data = []
-    for series in sofr_series:
-        data_df = fetch_and_process_data(fred, {'data_ref': series}, col_date)
-        sofr_data = update_sofr_data(sofr_data, data_df, col_date, series)
-        audit_data.append(collect_audit_info(data_df, series, series))
-        print(f"# {series}: Updated")
-        time.sleep(1)
-
-    sofr_data = sofr_data.sort_values(col_date).ffill()
-    save_combined_data(sofr_data, dataPath, "sofr/combined_sofr_data", SAVE_AS_PICKLE)
+def process_combined_dataset(data_type, data_info, fred, col_date, dataPath, SAVE_AS_PICKLE, PUSH_TO_GCP, bucket):
```

Ruff (0.8.2)

Missing return type annotation for public function `process_combined_dataset`

(ANN201)